### PR TITLE
bike project

### PR DESCRIPTION
**Pull request by yuron-maker**
Sorry. I did not notice that it was necessary to do block by block
[DEMO LINK](https://yuron-maker.github.io/layout_miami/)

### CODE REVIEW

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
https://github.com/mate-academy/layout_miami/blob/master/checklist.md
fix according checklist points:
1, 4, 8, 9. 10, 14, 15

also form submitting shouldn't change the link
check if all your styles according to mockup
image

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Fix previous comment
image

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,316 @@
...
+                </div>
+              </div>
+              <div class="grid__item--full">
+                <h3 class="details__subtitle">Auto Unlock</h3>
+                <p class="details__text">
+                  The app senses when you're nearby to unlock automatically. GPS
+                  tracking so you know where your bike is and can track it
+                  anytime.
+                </p>
+              </div>
```
Comment (SerhiiKirik) [inline src/index.html:188]:
```suggestion
      </section>
      
      <section class="details" id="details">
        <div class="container">
          <h2 class="page__section-title">The details</h2>
          
          <div class="details__content">
            <div class="grid details__section">
              <div class="grid__item--desktop-1-8 grid__item--tablet-1-4">
                <div class="details__photo-container">
                  <img
                    src="./images/photos/image 12.png"
                    alt="image12"
                    class="details__photo"
                  >
                </div>
              </div>
              
              <div class="grid__item--desktop-9-12 grid__item--tablet-5-6">
                <div class="details__photo-container">
                  <img
                    src="./images/photos/image 8.png"
                    alt="image8"
                    class="details__photo"
                  >
                </div>
              </div>
              
              <div class="grid__item--full">
                <h3 class="details__subtitle">Auto Unlock</h3>
                
                <p class="details__text">
                  The app senses when you're nearby to unlock automatically. GPS
                  tracking so you know where your bike is and can track it
                  anytime.
                </p>
              </div>
              
```

File: src/styles/blocks/_card.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    line-height: 140%;
+    color: #fff;
+  }
+
+  &__text {
+    margin-top: 8px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 140%;
+    color: #fff;
```
Comment (SerhiiKirik) [inline src/styles/blocks/_card.scss:39]: create variables for all colors

```suggestion
    color: $c-white;
```

**Review by SerhiiKirik — APPROVED** [review]
Good job 🔥 

but i recommend you use scroll-behavior: smooth; in html tag
https://user-images.githubusercontent.com/91411692/222194819-878e533a-75ef-4b26-bc02-d40c11102979.mov

**Pull request closed without merging**